### ISSUE TITLE
Crash on Launch

### ISSUE THREAD

**Issue by CozyMiel** [katubug/CottageWitch119#32]
**What Cottage Witch version are you using?**
Version: 1.10 and 1.14

**What were you doing when you crashed?**
Game crashes on launch. It worked perfectly fine last night and then this morning it just won't. 

**Screenshots**
![image](https://user-images.githubusercontent.com/131920432/234709633-49554596-fbf2-4821-b506-853900c3205d.png)

**Crash Reports or Latest.log**
1.10:  [hs_err_pid9072.log](https://github.com/katubug/CottageWitch119/files/11337525/hs_err_pid9072.log)

1.14:  [hs_err_pid14112.log](https://github.com/katubug/CottageWitch119/files/11337608/hs_err_pid14112.log)

**Comment by katubug** [comment]
Hi, I'm not entirely sure what's causing this, but I would recommend
updating to the latest version of the modpack (1.14.1b) via the Curseforge
launcher (or whatever launcher you use).

**Comment by CozyMiel** [comment]
I actually did try that newest version of the modpack as well and the same thing is happening. Thoughts?

**Comment by katubug** [comment]
Honestly, I'm not sure, except that it isn't happening globally so it could
be a driver or hardware issue.

I had another user report that they were able to launch by removing all the
resourcepacks and then placing them back in afterwards. And another user
said all they needed to do was update their video card drivers. Maybe one
of those will help?

**Comment by CozyMiel** [comment]
The things I've tried so far are:
-Updating my drivers
-checking through all the mods to see what's affecting it (the game still crashed even with all of them off)
-uninstalling and reinstalling the mod
-uninstalling and reinstalling curseforge
-uninstalling and reinstalling minecraft
-moving minecraft and curseforge to a new drive on my PC

I'm not particularly code-savvy, but I can do a lot of the basics. But nothing has worked so far... :/

**Comment by CozyMiel** [comment]
Latest Log: [latest.log](https://github.com/katubug/CottageWitch119/files/11362611/latest.log)
New Crash report: [hs_err_pid25928.log](https://github.com/katubug/CottageWitch119/files/11362613/hs_err_pid25928.log)

**Comment by katubug** [comment]
When you say the game crashes even with all the mods turned off, do you
mean with every single mod disabled at the same time? Or only disabling one
at a time?

I was going to suggest disabling Oculus, but if you've already tried that,
then obviously that's not helpful.

Other people have said that they had luck by disabling all the
resourcepacks and then reenabling them in-game. That's my last suggestion
though, this error seems to be only affecting a subset of users, but not me
so I can't test. You've done basically everything I'd have suggested.

My last recommendation is asking over at the moddedmc discord -
https://discord.gg/moddedmc in the user-help channel.

Hopefully the next update will fix whatever this is, but in the meantime if
you're able to fix it, please let me know what you did so I can suggest it
to others!

**Comment by CozyMiel** [comment]
Yes, All of them off at the same time. I was going to enable them again one at a time. But since the whole thing crashed without any of them on I wasn't sure how to proceed.

I can try that with the resource packs. I've even tried going back to older versions and they don't work either. 

I'll be sure to check out that discord too, I really enjoy this modpack its just chill enough and right up my alley, so I've been pretty bummed that I haven't been able to play. I also appreciate the time you've spent tryign to help, thank you~ <.<
So of course, if I do manage to figure it out I'll let you know what I did.

**Comment by CozyMiel** [comment]
I think it has something to do with Forge itself. I say that because Fabric works just fine. I tried creating a new profile of my own and I got the same error when I opened it using Forge--I made it a point not to install any mods yet. So it's information, although I'm not sure how helpful it is.

**Comment by CozyMiel** [comment]
I have good news! I got help from the CurseForge Discord, and for me it WAS in fact a graphics driver issue. I needed to update it and the way I did it the first time was just wrong. They gave me some links to go to in order to update it (I have an AMD). I can pop them in here but I'd definitely recommend that that if any of your suggestions don't work either, to ask them.

"Updating graphics card drivers
Outdated graphics card drivers can cause graphical issues and crashes.
Update your graphics card drivers using the tools provided by your graphics card brand.
-INTEL: https://www.intel.com/content/www/us/en/support/detect.html
-AMD: https://www.amd.com/en/support
-NVIDIA: https://www.nvidia.com/download/index.aspx
"

So I'm sorry for all the stress over the past couple of weeks, but I hope this helps.
